### PR TITLE
Update term definitions according to the core vocab

### LINKED ISSUE #182: Update term definitions to match the VCDM's vocab
Term definitions should be updated so that _this specification_ is the one defining its own terms. This change is reflected in a currently outstanding PR by @iherman https://github.com/w3c/vc-data-model/pull/1209/files#diff-dfe00db9d510a45f71289312990ee36df870f36c02094551bfba625dcd0d8e96R34

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #182

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/185.html" title="Last updated on Jul 26, 2023, 7:16 PM UTC (313dc88)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/185/2743534...313dc88.html" title="Last updated on Jul 26, 2023, 7:16 PM UTC (313dc88)">Diff</a>

### CODE REVIEW

**Review by andresuribe87 — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
Seems like the term definitions should be to the vocabulary.

Cc@iherman

**Review by iherman — CHANGES_REQUESTED** [review]
Unfortunately, that is incorrect... I propose to change the sentences as follows:

```html
<p>
This term is part of the 
<a href="https://www.w3.org/ns/credentials#JsonSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
</p>
…
<p>
This term is part of the 
<a href="https://www.w3.org/ns/credentials#VerifiableCredentialSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
</p>
```

I do not believe referring to the JSON-LD spec's term definition brings anything here, so I removed that reference, too.

Of course, the URL-s must be updated if the final term names change.

(GitHub does not let me propose a direct edit to the full sentence, hence this comment.)

File: index.html [hunk 1]
```diff
@@ -155,7 +155,7 @@ <h2>Data Model</h2>
             <h3>JsonSchema2023</h3>
             <p>
                This <a href="https://www.w3.org/TR/json-ld/#dfn-term-definition">term definition</a>
-               is <code>https://www.w3.org/TR/vc-json-schema/#JsonSchema2023</code>.
+               is <code>https://www.w3.org/TR/vc-json-schema#JsonSchema2023</code>.
```
Comment (TallTed) [inline index.html:158]: The slashless URI gets a `302 Found` redirect to the slashed URI, where the fragment does not resolve.

The text found in that document always reads as the mixed case `JsonSchema2023`, but the fragment identifier for the section where it is defined is the lowercase [`#jsonschema2023`](https://www.w3.org/TR/vc-json-schema/#jsonschema2023), where it says --
> This [term definition](https://www.w3.org/TR/json-ld/#dfn-term-definition) is https://www.w3.org/ns/credentials#JsonSchema2023.

-- which gets a `302 Found` redirect to `https://www.w3.org/ns/credentials/` which gets a `404 Not Found`. 

This is confusing to me, and I've been working with all this stuff for quite a while. It seems that anyone new to any of this material who encounters the above will quickly throw up their hands and walk away.

File: index.html [hunk 2]
```diff
@@ -238,7 +238,7 @@ <h3>JsonSchema2023</h3>
             <h3>VerifiableCredentialSchema2023</h3>
             <p>
                This <a href="https://www.w3.org/TR/json-ld/#dfn-term-definition">term definition</a>
-               is <code>https://www.w3.org/TR/vc-json-schema/#VerifiableCredentialSchema2023</code>.
+               is <code>https://www.w3.org/TR/vc-json-schema#VerifiableCredentialSchema2023</code>.
```
Comment (TallTed) [inline index.html:241]: Similar to comments above on `JsonSchema2023`.

**Comment by decentralgabe** [comment]
updated based on @iherman's comment.

@TallTed I notice the 404 as well, but I trust @iherman's suggestion. Maybe it'll be resolved after the core data model PR is merged?

**Comment by iherman** [comment]
Sorry @TallTed and @decentralgabe I screwed up. The right URLs should be

- https://www.w3.org/2018/credentials/#JsonSchema2023
- https://www.w3.org/2018/credentials/#VerifiableCredentialSchema2023

ie:

```html
<p>
This term is part of the 
<a href="https://www.w3.org/2018/credentials/#JsonSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
</p>
…
<p>
This term is part of the 
<a href="https://www.w3.org/2018/credentials/#VerifiableCredentialSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
</p>
```

Sorry about the confusion.

**Comment by decentralgabe** [comment]
updated

File: index.html [hunk 3]
```diff
@@ -246,9 +246,8 @@ <h3>JsonSchema2023</h3>
          <section>
             <h3>JsonSchemaCredential2023</h3>
             <p>
-               This <a href="https://www.w3.org/TR/json-ld/#dfn-term-definition">term definition</a>
-               is <code>https://www.w3.org/ns/credentials#JsonSchemaCredential2023</code>.
-            </p>
+               This term is part of the 
+               <a href="https://www.w3.org/2018/credentials/#VerifiableCredentialSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
```
Comment (decentralgabe) [inline index.html:250]:
```suggestion
               <a href="https://www.w3.org/2018/credentials/#VerifiableCredentialSchema2023">Verifiable Credentials Vocabulary v2.0</a>.
             </p>
```

File: index.html [hunk 4]
```diff
@@ -246,9 +246,9 @@ <h3>JsonSchema2023</h3>
...
+             </p>
```
Comment (decentralgabe) [inline index.html:251]:
```suggestion
            </p>
```

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
@OR13 
> Seems like the term definitions should be to the vocabulary.
> 
> Cc@iherman

@OR13, @decentralgabe, also with https://github.com/w3c/vc-data-model/pull/1218, I believe this PR can be merged.

**Review by OR13 — APPROVED** [review]

**Pull request merged**